### PR TITLE
add delete photo business logic on FE

### PR DESCRIPTION
**Pull request by rostyk1331**

### CODE REVIEW

File: client/src/components/Cropper/Cropper.tsx [hunk 1]
```diff
@@ -2,8 +2,10 @@ import React from 'react';
 import Avatar from 'react-avatar-edit';
 import noAvatar from '../assets/noAvatar.svg';
 import {Button} from 'reactstrap';
-
-class Cropper extends React.Component {
+interface IFormProps {
```
Comment (InnaKutsik) [inline client/src/components/Cropper/Cropper.tsx]: do we have form?

File: client/src/store/profile/reducers.ts [hunk 2]
```diff
@@ -47,6 +49,14 @@ export const profileReducer = (
             error: action.payload,
             loaded: false,
         };
+    case DELETE_PHOTO_SUCCESS:
```
Comment (InnaKutsik) [inline client/src/store/profile/reducers.ts]: no DELETE_PHOTO_PENDING, DELETE_PHOTO_ERROR

без лоудера, щоб юзер не клікав по 100-500 раз?

Comment (rostyk1331) [reply client/src/store/profile/reducers.ts]: Done.

File: client/src/components/Cropper/Cropper.tsx (hunk 1 above)
Comment (rostyk1331) [reply client/src/components/Cropper/Cropper.tsx]: Fiexed.

Comment (rostyk1331) [reply client/src/components/Cropper/Cropper.tsx]: https://github.com/jsgram/JSGram/pull/75

File: client/src/store/profile/reducers.ts (hunk 2 above)
Comment (rostyk1331) [reply client/src/store/profile/reducers.ts]: https://github.com/jsgram/JSGram/pull/75

**Pull request closed without merging**